src/Controller/ParticipantController.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\Participant;
+use App\Model\Tournament;
+use App\Services\ParticipantService;
+use App\Services\TournamentService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ParticipantController extends AbstractController
+{
+    private ParticipantService $participantService;
+    private TournamentService $tournamentService;
+
+    public function __construct(ParticipantService $participantService, TournamentService $tournamentService)
+    {
+        $this->participantService = $participantService;
+        $this->tournamentService = $tournamentService;
+    }
+
+    /**
+     * @Route("/api/tournaments/{tournamentId}/participants", name="add_participant", methods={"POST"})
+     */
+    public function addParticipant(Request $request, string $tournamentId): Response
+    {
+        $parametersAsArray = json_decode($request->getContent(), true);
```

Comment:
vous pouvez utiliser le validateur de Symfony pour checker les champs et y appliquer des contraintes => 

https://symfony.com/doc/current/validation.html#using-the-validator-service
https://symfony.com/doc/current/reference/constraints.html